controllers/publishController.js

```diff
@@ -7,23 +7,27 @@ module.exports = {
   publish (publishParams, fileName, fileType) {
     return new Promise((resolve, reject) => {
       let publishResults = {};
+      let file;
+      let claim;
       // 1. make sure the name is available
-      publishHelpers.checkNameAvailability(publishParams.name)
+      publishHelpers.checkClaimNameAvailability(publishParams.name)
       // 2. publish the file
       .then(result => {
         if (result === true) {
           return lbryApi.publishClaim(publishParams);
         } else {
-          return new Error('That name has already been claimed by spee.ch.  Please choose a new claim name.');
+          return new Error('That name has already been claimed by spee.ch.');
```

Comment:
"This name is already in use by spee.ch"

"Claim" is an unusual word that is very LBRY specific. Let's avoid using it until we have to.

Reply:
:+1: